app/controllers/concerns/sms.rb

```diff
@@ -2,36 +2,16 @@
 require "twilio-ruby"
 
 # A set of utility methods used in relation to SMS notifications
-# @author Eric Schlange <eric.schlange@northwestern.edu>
 module Sms
-  extend ActiveSupport::Concern
-
-  # Send an SMS message body to a specified receiving participant (leaving
-  # off recipient defaults the value to the current participant)
-  # rubocop:disable Metrics/AbcSize
-  def send_sms(recipient = current_user, message_body)
+  def send_sms(recipient, message_body)
+    return unless Rails.env.staging? || Rails.env.production?
```

Comment:
@michaelwehrley I'm trying to say that there should only be two possible returns from a Ruby method. One at the beginning of the method to quickly exit if some condition is not met, the other is the implicit Ruby, last statement return. 

In this example something like this would suffice: 
```return unless (Rails.env.staging? || Rails.env.production? ) && !recipient.phone_number.blank?```

Or, if that is deemed too complex for a condition, it can be encapsulated into a private routine:

```return unless valid_environment_and_recipient_configuration(recipient)```
```
def valid_environment_and_recipient_configuration(recipient)
  (Rails.env.staging? || Rails.env.production? ) && !recipient.phone_number.blank?
end
```